packages/kyt-core/config/webpack.base.js

```diff
@@ -5,75 +5,104 @@
 
 const path = require('path');
 const webpack = require('webpack');
+const shell = require('shelljs');
 const autoprefixer = require('autoprefixer');
-const babel = require('./babel');
-const { buildPath, userNodeModulesPath } = require('kyt-utils/paths')();
+const { buildPath, userNodeModulesPath, userBabelrcPath } = require('kyt-utils/paths')();
+const logger = require('kyt-utils/logger');
 
-module.exports = options => ({
-  node: {
-    __dirname: true,
-    __filename: true,
-  },
+module.exports = (options) => {
+  const hasBabelrc = shell.test('-f', userBabelrcPath);
+  if (!hasBabelrc) {
+    logger.warn('No user .babelrc found. Using kyt default babel preset...');
+  }
```

Comment:
Just checking these are the only changes you made in this file yah? The rest of the diff just looks like spacing